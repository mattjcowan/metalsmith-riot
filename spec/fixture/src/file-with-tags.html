<!DOCTYPE html>
<html dir="ltr" lang="en-US">
    <head></head>
    <body>
        <section>
            <div data-is="test" active="false">WITH SOME CONTENT</div>
            <div data-is="test" active="true"></div>
            <test active="true">MORE CONTENT</test>
            <test active="false">
                <test active="true">MORE CONTENT</test>
            </test>
        </section>
    </body>
</html>