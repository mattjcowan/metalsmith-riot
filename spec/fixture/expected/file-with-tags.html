<!DOCTYPE html>
<html dir="ltr" lang="en-US">
  <head></head>
  <body>
    <section>
      <div data-is="test" active="false"><span data-awe="NOT AWESOME">this is text</span> WITH SOME CONTENT</div>
      <div data-is="test" active="true"><span class="active" data-awe="AWESOME">this is text</span> </div>
      <test active="true"><span class="active" data-awe="AWESOME">this is text</span> MORE CONTENT</test>
      <test active="false"><span data-awe="NOT AWESOME">this is text</span>
        <test active="true"><span class="active" data-awe="AWESOME">this is text</span> MORE CONTENT</test>
      </test>
    </section>
  </body>
</html>